src/lang/en-US.json

```diff
@@ -433,13 +433,18 @@
   "subheading_team_elo_rankings": "k=32, init=1000",
   "hero_pro_tab": "Professional",
   "hero_public_tab": "Public",
+  "hero_turbo_tab": "Turbo",
```

Comment:
Pretty sure there is already a string for "Turbo".

Reply:
I'm fine with this being separate.